### PR TITLE
Added load-balancer hostname/IP to ingress details

### LINKED ISSUE #83: GKE ingress IP not visible in pane
Lens

### PR DESCRIPTION
**Pull request by jim-docker**
Signed-off-by: Jim Ehrismann <jehrismann@mirantis.com>

fixes #83

### CODE REVIEW

**Review by Nokel81 — APPROVED** [review]

**Comment by Nokel81** [comment]
Only I don't know why there are changes to the `.po` files

**Comment by jim-docker** [comment]
> Only I don't know why there are changes to the `.po` files

There are some new internationalizations here. I assumed since the `.po` files are in the repo they need to be updated(?)

**Comment by Nokel81** [comment]
Yes, I just didn't notice any new int8s.

**Comment by jim-docker** [comment]
> #response_container_BBPPID{font-family: initial; font-size:initial; color: initial;} Yes, I just didn't notice any new int8s 

`<Trans/>` elements

File: locales/en/messages.po [hunk 1]
```diff
@@ -33,6 +33,10 @@ msgstr "<0>Your browser does not support all Lens features. </0> Please consider
 msgid "<0>{0}</0> successfully created"
 msgstr "<0>{0}</0> successfully created"
 
+#: src/renderer/components/input/input.validators.ts:40
```
Comment (Nokel81) [inline locales/en/messages.po:36]: Namely this one

**Pull request merged**